### PR TITLE
Added information on fields for storing DNA methylation information

### PR DESCRIPTION
**Pull request by heathsc**
Added a description of new fields that allow the storing of DNA methylation information to be stored with DNA sequencing variation in a VCF file.  This is important to allow the results of bisulfite sequencing experiments which generate both sequence variation and DNA methylation information to be stored in a standard format.

These proposed changes were originally submitted as a new format, and are now being resubmitted as a modification of the VCFv4.3 standard following a suggestion from the original pull request.

### CODE REVIEW

File: VCFv4.3.tex [hunk 1]
```diff
@@ -31,16 +31,21 @@
...
-be used instead.
-In order to ensure interoperability across platforms, VCF compliant implementations must support
-both LF (\texttt{\textbackslash n}) and CR+LF (\texttt{\textbackslash r\textbackslash n}) newline conventions.  
+line (prefixed with "\#"), and data lines each containing information
+about a position in the genome and genotype information on samples for
+each position (text fields separated by tabs). The VCF format can also
+store information on DNA methylation from bisulfite sequencing
+experiments and other sources alongside information about genome
+sequence variation.  Zero length fields are not allowed, a dot (".")
+should be used instead. In order to ensure interoperability across
```
Comment (yfarjoun) [inline VCFv4.3.tex]: should? MUST!

**Comment by heathsc** [comment]
No problem changing ‘should’ to ‘must’.  Do I have to submit a new pull request to do this?

Simon

**Comment by cyenyxe** [comment]
It is not necessary to create a new pull request, but please rebase your branch against master to ensure that conflicts are resolved. Then you can safely replace "should" with "must".

**Comment by mp15** [comment]
@heathsc It also helps if when you create future pull requests you create it on a branch and then tick the box on the pull request allowing allowing maintainers with write access to commit to your branch.

**Comment by cyenyxe** [comment]
@MarcosFernandez, could you please rebase these changes against the master branch? As you can see on this page, there are several files with conflicts, even though they are not related with the VCF specification.

**Comment by heathsc** [comment]
Conflicts are already solved. Is it able to merge??

**Comment by yfarjoun** [comment]
this is a bit of a stal PR. if there's still interest in it, please rebase on master and re-open.

**Comment by heathsc** [comment]
There is certainly interest, however we never got any feedback after the previous attempt.  The approval process does appear to be very opaque!  For IHEC it is important that the methylation specific tags are documented officially as part of the VCF spec - so how should we proceed to make this happen?

Simon

**Comment by yfarjoun** [comment]
you have comments given to you on this PR. please respond to them. you do not need to open a new PR, rebase this one on top of the master branch:

```bash 
$ git fetch
$ git rebase origin/master
```

since there are conflicts you'll probably have to tidy them up (https://help.github.com/en/articles/resolving-merge-conflicts-after-a-git-rebase)


then you'll need to force push
```
git push -f
```

In addition to rebasing there were particular requests, so please respond to them, commit and re-push your changes and they will be reflected in this PR. Sorry for the delay in response.

**Comment by d-cameron** [comment]
Base modification support has been added to VCFv4.5. There is a public consultation period before the specifications are finalised so please review the proposed design and highlight any issues/limitations in the proposed model as the encoding is quite different to the one proposed in this PR.

**Pull request closed without merging**